### ISSUE TITLE
search all fields: municipality

### ISSUE THREAD

**Issue by DimEvil** [gbif/portal-feedback#4804]
Can https://dwc.tdwg.org/terms/#dwc:municipality be added in the advance occurrence search? 

The main reason is that is possibly very interesting for local administrations

**Comment by MortenHofft** [comment]
The website is simply using the public APIs. So first it would have to be added to the API.
Either using the published verbatim field `dwc:municipality` or using the coordinates and the aligned GADM fields
https://www.gbif.org/occurrence/map?occurrence_status=present&gadm_gid=BEL.2.1.1_1 . 
For Belgium for example we stop at arrondissements and do not include municipalities.

The quality of using coordinates and GADM over the `dwc:municipality` is that we do not rely on publishers filling in the information. And do not have issues with different spellings, abbreviations etc.

**Comment by DimEvil** [comment]
Indeed, GADM municipalities for Belgium are not available (it would be even a better solution to add these). 

A problem is that somethimes the high resolution coordinates of an occurrence are not in the coresponding municipality (Gridded data and centroids) and this would mean a possible difference between GADM and dwc:municipality (not that I worry too much about that)

so if GADM level 3 for Belgium would become available, It would solve already many problems.

**Comment by MattBlissett** [comment]
Municipalities for Belgium do exist in GADM, but they are a level 4 division and we are only adding up to level 3 in GBIF interpretation:

![image](https://github.com/gbif/portal-feedback/assets/583320/415fd6f9-ab58-4f7b-89d5-3ad4e769a717)

We'll need to consider the speed and potential data volume of adding level 4 divisions.

**Comment by DimEvil** [comment]
ok, level 4 :)
It would be of great benefit

or having dwc:municipality in the advanced search. (Coming with all the global misspellings of course...)

**Comment by AbbyBenson** [comment]
I'm not sure but will this issue come in to play? https://github.com/gbif/portal-feedback/issues/3044
